bootstrap.sh

```diff
@@ -29,6 +29,6 @@ if [ ! -z "$XCS_BOT_ID"  ]; then
 fi
 
 if [ ! -f Carthage/Cartfile.resolved ]; then
-  carthage bootstrap $CARTHAGE_VERBOSE --platform ios --color auto --no-use-binaries
+  carthage bootstrap $CARTHAGE_VERBOSE --platform ios --color auto
```

Comment:
I'm assuming the reason Leanplum isn't able to get pulled in through Carthage without this is because they haven't put their source up yet to build from - only the frameworks as releases. I don't know if there is a per-dependency way of doing this check.